### PR TITLE
Feature/df1557 replace uuids

### PR DESCRIPTION
**Pull request by fabiangotzen**

### CODE REVIEW

File: reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/model/IdMapping.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package ch.admin.seco.service.reference.service.mapping.model;
+
+public class IdMapping {
+
+    private String UUID;
```
Comment (cjpsycos) [inline reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/model/IdMapping.java]: what kind of id is this? oldOccupationId or houseId

File: reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/model/IdMapping.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    private String compositeKey;
```
Comment (cjpsycos) [inline reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/model/IdMapping.java]: is this a key or the newOccupationCompositeId

File: reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/model/IdMapping.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package ch.admin.seco.service.reference.service.mapping.model;
+
+public class IdMapping {
```
Comment (cjpsycos) [inline reference-app/src/main/java/ch/admin/seco/service/reference/service/mapping/model/IdMapping.java]: is this the mapper for the occupationLabelIds? I see something like generateOccupationLabelIdMappingFromFile, so this should be called OccupationLabelIdMapping

**Comment by fabiangotzen** [comment]
Will probably not be committed -> closing this for the moment.

**Pull request closed without merging**